### ISSUE TITLE
内核更新后，opkg所有软件包都不能被以前版本使用

### ISSUE THREAD

**Issue by jpsn123** [kiddin9/OpenWrt_x86-r2s-r4s-r5s-N1#1514]
https://op.supes.top里面的包都会更新到新的内核，但是上个版本的就能使用这里的软件了，下载软件的时候会出错。
可以不可以在路径中加入内核的区别，不然你这里内核升级了，就会逼着所有人升级到最新版本。

**Comment by hb-0** [comment]
我发现一直无法更新列表，打开https://op.supes.top/targets/x86/64/5.15.86
一看，好嘛，404了
把配置改成最新的，安装软件包时又报" cannot find dependency kernel (= 5.15.127-1-1) for ..."的问题

是要升级内核吗？
要如何升级内核呢，不会是重新下载最新的定制镜像吧？？？

**Comment by jpsn123** [comment]
@hb-0  是的 ，目前来说我就是这样解决，一开始我以为作者自己弄成这样，后来才知道openwrt官方就是这样子的，因为目前23还不是稳定版本，所以经常更新内核，等以后稳定了就好了
